src/desktop/config.coffee

```diff
@@ -66,6 +66,7 @@ module.exports =
   GOOGLE_ADWORDS_ID: null
   GOOGLE_MAPS_API_KEY: null
   HASHTAG_LAB_ADS_ALLOWLIST: ''
+  HASHTAG_LAB_ADS_ENABLED: ''
```

Comment:
Should this default to a boolean value of `false` like other _ENABLED configuration items defined here?